README.md

```diff
@@ -1,2 +1,16 @@
-# hw01
-Repository for homework 01
+#Vivek Ramakrishnan
```

Comment:
I had the same issue with this. On Rstudio's preview it likely worked, but the view on github shows #Vivek Ramakrishan instead of a heading. I guess you need a space after the # to not have this issue